### ISSUE TITLE
Integration with tabbed

### ISSUE THREAD

**Issue by rsvdr** [pwmt/girara#49]
It would be nice to have better integration with suckless [tabbed](https://tools.suckless.org/tabbed/).
There is a [current issue](https://github.com/pwmt/zathura/issues/432) with the `render_all()` function queuing resizes.
This prevents `render_job` from running when using zathura+tabbed, it runs only after tabs are switched.

**Comment by MaMiMa81** [comment]
> It would be nice to have better integration with suckless [tabbed](https://tools.suckless.org/tabbed/). There is a [current issue](https://github.com/pwmt/zathura/issues/432) with the `render_all()` function queuing resizes. This prevents `render_job()` from running when using zathura+tabbed, it runs only after tabs (or windows) are switched.

Yes I too would appreciate that - it was working well with tabbed using zathura v. 0.5.2 with girara v. 0.4.0 & zathura-pdf-poppler v. 0.3.1 as I posted over in the thread:
[https://github.com/jarun/nnn/issues/1793](url)

**Comment by MaMiMa81** [comment]
@rsvdr

This may be of some help to you

[https://github.com/jarun/nnn/issues/1793#issuecomment-2541621428](url)

**Comment by eylles** [comment]
Seems the regression was introduced on version 4.3 or 4.4 of libgirara as from what i've read some users have been able to build zathura with libgirara 4.2 and get the proper behaviour with tabbed.

**Comment by sebastinas** [comment]
Closing this. Somebody who understands tabbed and window embedding needs to take a look at this in zathura.

**Comment by MaMiMa81** [comment]
For those using `nnn` file manager with `preview-tabbed` plugin & wishing to use `zathura` with it I have updated my amendments to this plugin:

[nnn](https://github.com/jarun/nnn/issues/2126#issue-4269658268)
